STM32F405RGTx/Core/Inc/mc.hpp

```diff
@@ -10,15 +10,20 @@ class MC {
         float pVa;
         float pVb;
         float pVc;
+        
         // DC battery voltage
         float dc_voltage;
+        
         // Phase currents
         float pIa;
         float pIb;
         float pIc;
-        // MOSFET and DC link capacitor temperatures
+        
+        // MOSFET
         float fet_temps[3];
+        // Note: dc_cap_temp dne for Powerboard rev 2 but will for the next rev
         float dc_cap_temp;
+        
         // Current position, speed, and acceleration
         float curr_pos;
         float curr_speed;
```

Comment:
I know this isn't in the class diagram, but maybe it's a good idea to have two different variables for speed: 1 for speed measured by the IMU, and another for the speed measured by the ring encoder?

Reply:
I was thinking that I'll add that once we also have the accelerometer code ready